### ISSUE TITLE
check_whois.pl allow NS check to be a regex

### ISSUE THREAD

**Issue by jeff-cook** [HariSekhon/nagios-plugins#146]
Currently the check_whois.pl check will match the nameservers to an expected list.  It would be great if this could be a regex expect line the check_dns.pl check allows.  

Thanks

**Comment by HariSekhon** [comment]
Good idea, I'll look at implementing this but it'll take a while as I'm on a client engagement right now.
